app/routes/users/components/UserProfile.js

```diff
@@ -177,7 +181,14 @@ const ListEvents = ({ events, noEventsMessage, loggedIn }: EventsProps) => (
   </div>
 );
 
-export default class UserProfile extends Component<Props, EventsProps> {
+type UserProfileState = { showAbaId: boolean };
+
+export default class UserProfile extends Component<Props, UserProfileState> {
+  constructor(props: Props) {
```

Comment:
Bold of you to not rewrite this to a functional component, but okay

Reply:
Yeah, that was kinda lazy. Rewrote it now. ✌🏻